portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,25 +24,44 @@
 import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+@WebServlet("/comments")
 public class DataServlet extends HttpServlet {
 
-    ArrayList<String> mylist = new ArrayList<String>();
-        
-
-    public DataServlet(){
-        mylist.add("Hello this");
-        mylist.add("is a");
-        mylist.add("test");
-    }
-    
+    ArrayList<String> commentlist = new ArrayList<String>();
+    ArrayList<String> comment = new ArrayList<String>();
```

Comment:
Why do you need both commentList and comment? It seems to me like having a Comment class and appending to the commentList would accomplish the same thing, without having to maintain the comment state across the get and post.

Reply:
I didn't know you could use classes in gson conversion.

Reply:
You should be able to do something like this: https://github.com/google/gson/blob/master/UserGuide.md#object-examples

Reply:
Finished